File: app/src/main/java/com/nagpal/shivam/vtucslab/Model/LabExperiment.java
package com.nagpal.shivam.vtucslab.Model;

public class LabExperiment {

    private String serialOrder;
    private LabExperimentSubPart[] labExperimentSubParts;

    public String getSerialOrder() {
        return serialOrder;
    }

    public LabExperiment setSerialOrder(String serialOrder) {
        this.serialOrder = serialOrder;
        return this;
    }

    public LabExperimentSubPart[] getLabExperimentSubParts() {
        return labExperimentSubParts;
    }

    public LabExperiment setLabExperimentSubParts(LabExperimentSubPart[] labExperimentSubParts) {
        this.labExperimentSubParts = labExperimentSubParts;
        return this;
    }
}

File: app/src/main/java/com/nagpal/shivam/vtucslab/models/LabExperiment.kt
package com.nagpal.shivam.vtucslab.models

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class LabExperiment(
    var serialOrder: String,
    var labExperimentSubParts: List<LabExperimentSubPart>,
)
